src/igz_controls/components/default-dropdown/default-dropdown.tpl.html

```diff
@@ -41,7 +41,7 @@
         </div>
     </div>
 
-    <div class="default-dropdown-container" tabindex="-1"
+    <div class="default-dropdown-container" data-ng-class="$ctrl.getAdditionalClass()" tabindex="-1"
```

Comment:
The `$ctrl.additionalClass` binding is textual (`'@'`) so you can just put it there in the `class` attribute, like this:
```suggestion
    <div class="default-dropdown-container {{$ctrl.additionalClass}}" tabindex="-1"
```
Then there's no need for the public method `getAdditionalClass()`.

Reply:
Also, I will change it, because of possible performance issues. 'getAdditionalClass()' calls each time on mouse move, so I think if we could avoid it, better to do it.